xbanish.c

```diff
@@ -15,16 +15,17 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
+#define _POSIX_C_SOURCE 2
```

Comment:
IMO it's better to define feature test macros in the Makefile under `CPPFLAGS` variable. But since it's a single file source, probably doesn't matter much.